### PR TITLE
Adding parcel and removing firebase config

### PR DESCRIPTION
**Pull request by Rafi993**
I added parcel so as to enable easy reloading during dev and to minify code in production. I have remove Firebase config and separated it out. It is always good not to push your API keys into codebase

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,13 +1,7 @@
+import config from "./config";
+
 (function() {
   // Initialize Firebase
```
Comment (sbimochan) [inline app.js:4]: This doesn't need to be secured according to firebase but yes its better to isolate it out.

**Review by sbimochan — APPROVED** [review]

File: app.js (hunk 1 above)
Comment (Rafi993) [reply app.js:4]: Yes it does not but it is better to not to have API keys in the repo

Comment (Rafi993) [reply app.js:4]: since others who work on the fork will use their own project

**Comment by sbimochan** [comment]
Thanks for the PR. Since this is a light vanilla project. I don't think it needs to be minified or need to have CI/CD setup. So I've reverted it. Other PRs are welcome.
Thanks! Happy hacktober!

**Comment by Rafi993** [comment]
Minification is something you get with parcel without doing anything. The main advantage of using parcel is es6 imports and hot reloading when you are working rather than refreshing manually. Parcel is a dev dependency not a direct dependency.

**Pull request merged**